index.html

```diff
@@ -153,7 +156,7 @@ <h1>About me</h1>
         </section>
         <section id="form-container">
             <div id="form-pargrah">
-                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
             </div>
             <form action="https://formspree.io/f/xknaybwq" method="post">
                 <ul>
```

Comment:
- [OPTIONAL] I would implore you to work on this category so that there would be sufficient space as it is on Figma.
### Your project
![image](https://user-images.githubusercontent.com/86200356/236102980-461c8f66-626b-46ef-8b8c-629c88a3e18f.png)

### Figma 
![image](https://user-images.githubusercontent.com/86200356/236103591-f9f5d6e5-aabd-4791-99b6-e509cc03a7c1.png)